gen/ground_forces/ai_ground_planner.py

```diff
@@ -80,6 +80,10 @@ def __init__(self, cp: ControlPoint, game):
 
     def plan_groundwar(self):
 
+        ground_unit_limit = self.cp.frontline_unit_count_limit
+
+        remaining_available_frontline_units = ground_unit_limit
```

Comment:
> > Needs to update `CAMPAIGN_FORMAT_VERSION`. This is a major version break.
> > Other than that and a few other notes, this is looking good 👍
> 
> To `5.0` or `4.3`?

Major, so 5.0.

Reply:
Part 2 here was fixed. I'd still like to hear from @Schneefl0cke (also pinged in Discord now), but since the procurer has been fixed I think the answer is just that the unit ratios need to be adhered to here as well, but you can't do that before the other patch is merged so I think you won that race :)

Reply:
You still buy units randomly, right? My changes just change the randomness and are not affected by the limit of units. At least thats what I see at first glance.

Reply:
@DanAlbert @SnappyComebacks  I pulled the changes and fixed it on my branch, but now there are really not a lot of frontline units. Just 8 or 9 when the FOB or airport is not big. This seems really low for me. Maybe its possible to increase the number of ground units supported by each ammo depot, idealy every user could change this in his settings at the start of the liberation cmpaign in the wizard.

Reply:
Was a problem on my side because of merge, fixed it.

Reply:
@DanAlbert @SnappyComebacks 
Sometimes the number of vehicles seems rather small, like 17 vs 13. I think it would be a great benefit, if it could be altered in the settings menu in the wizard, how many ground units are supported by each ammo depot.